tests/cleanup_dummy_device.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Please create a `tests/tasks` sub-directory for these common task files used by the tests.
I don't think we mention this in the best practices guide but we should . . .

Reply:
Makes sense, I will update it to follow this.